src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/util/CardValue.java

```diff
@@ -21,6 +21,9 @@ public class CardValue {
 
```

Comment:
ich würde hier die ordnung beibehalten, z.b. EIGHT_OF_SPADES hoch zu den SPADES und DIAMONDS in eine separate gruppe